LICENSE

```diff
@@ -1,340 +1,6 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 2, June 1991
+rippled-gateway-balances was made for use in conjunction with the QuickGatewayKit by Taylor Warden, 
+both packages are licensed under a Creative Commons Attribution 4.0 International License, please see 
```

Comment:
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.